modules/govuk/manifests/apps/email_alert_api/delivery_attempt_status_check.pp

```diff
@@ -0,0 +1,21 @@
+# == Define: govuk::apps::email_alert_api::delivery_attempt_status_check
+#
+# Creates an Icinga check which checks Graphite for high numbers of delivery
+# attempts with a specifc status.
+#
+# === Parameters
+#
+# [*title*]
+#   The status of the delivery attempts to look for.
+#
+define govuk::apps::email_alert_api::delivery_attempt_status_check() {
+  @@icinga::check::graphite { "email-alert-api-delivery-attempt-${title}":
+    host_name => $::fqdn,
+    target    => "sum(stats.govuk.app.email-alert-api.*.delivery_attempt.status.${title})",
+    warning   => '0.5',
+    critical  => '1',
+    from      => '1hour',
+    desc      => "High number of ${title} delivery attempts",
+    notes_url => monitoring_docs_url(email-alert-api-delivery-attempt-status),
```

Comment:
This page currently doesn't exist, but I'm going to add it as part of this task.

Reply:
https://github.com/alphagov/govuk-developer-docs/pull/2099